app/org/maproulette/framework/service/TaskReviewService.scala

```diff
@@ -31,4 +34,41 @@ class TaskReviewService @Inject() (repository: TaskReviewRepository) {
   def expireTaskReviews(duration: FiniteDuration): Int = {
     this.repository.expireTaskReviews(duration)
   }
+
+  /**
+    * Gets a list of tasks that have been reviewed (either by this user or requested by this user)
+    *
+    * @param user      The user executing the request
+    * @param reviewTasksType
+    * @param searchParameters
+    * @param startDate Limit tasks to reviewed after date (YYYY-MM-DD)
+    * @param endDate   Limit tasks to reviewed before date (YYYY-MM-DD)
+    * @return A list of tasks
+    */
+  def getReviewMetrics(
+      user: User,
+      reviewTasksType: Int,
+      searchParameters: SearchParameters,
+      mappers: Option[List[String]] = None,
+      reviewers: Option[List[String]] = None,
+      priorities: Option[List[Int]] = None,
+      startDate: String,
+      endDate: String,
+      onlySaved: Boolean = false,
+      excludeOtherReviewers: Boolean = false
+  ): ReviewMetrics = {
+    this.taskReviewDAL.getReviewMetrics(
```

Comment:
How hard would it be to move this particular function from the DAL to a new repository?